mapper/cumulocity/c8y_translator_lib/src/lib.rs

```diff
@@ -1,2 +1,5 @@
 pub mod json;
 pub mod serializer;
+#[cfg(test)]
+#[macro_use]
+extern crate assert_matches;
```

Comment:
You can move that under `mod tests`. Try to use just: 

```rust
use assert_matches::assert_matches;
```

Should not need `extern crate` and no `macro_use` anymore.

Reply:
ok, fixed